app/config/sql/user/find.sql

```diff
@@ -0,0 +1 @@
+SELECT * FROM users WHERE $1^=$2;
```

Comment:
What does $2 represent in this? I know $1^ reflects the users email, but does $2 reflect the inputted email during login?